waspc/data/Generator/templates/server/src/core/auth/prismaMiddleware.js

```diff
@@ -16,8 +16,8 @@ const registerUserEntityValidation = (prismaClient) => {
       if (['create', 'update', 'updateMany'].includes(params.action)) {
         validateUser(params.args.data, params.args, params.action)
       } else if (params.action === 'upsert') {
-        validateUser(params.args.create.data, params.args, 'create')
-        validateUser(params.args.update.data, params.args, 'update')
```

Comment:
These `upsert` updates are just because I was in this file. Prisma docs changed recommended use for `upsert` and it no longer passes a data property under `create`/`update`. Exe: https://www.prisma.io/docs/concepts/components/prisma-client/crud#update-or-create-records